src/lib/classifyquote.tsx

```diff
@@ -39,7 +45,7 @@ melanie
 [5:23 PM]
 I just got home, so I'm gonna read them now!
 */
-const slackHeaderRegex = /^(.+?[^\s])( |$\n?)\[\d{1,2}:\d{2} (A|P)M\](\s+)?/;
+const slackVerboseHeaderRegex = /^(.+?[^\s])( |$(\r\n|\n)?)\[\d{1,2}:\d{2} (A|P)M\](\s+)?/;
```

Comment:
I've been wondering: why do you have all these capturing groups for the parts of the regex you don't care about? Can't you just put the whitespace and square braces and such outside of the capturing groups so you only capture what you want? This applies to all the regexes.